bin/load_analytics_files_in_Solr.sh

```diff
@@ -22,6 +26,9 @@ for EXP_ID in $ACCESSIONS; do
   INPUT_JSONL=${analytics_jsonl_dir}/${EXP_ID}.jsonl SOLR_COLLECTION=bulk-analytics SCHEMA_VERSION=1 solr-jsonl-chunk-loader.sh  # SOLR_COLLECTION=bulk-analytics SOLR_PROCESSORS=dedupe
   if [ $? -ne 0 ]; then
     echo "Loading JSONL to $SOLR_HOST failed for $EXP_ID"
+    if [ -f $failed_accessions_output ]; then
+      echo $EXP_ID >> $failed_accessions_output
+    fi
```

Comment:
This will only happen if there is a file in the file system in the path stored for `$failed_accessions_output`. If such variable was not defined in the first place, then you won't find a file either. The file was only created as well if the variable was defined. Makes sense? The complicated logic really comes from the fact that Snakemake will be re-sending this to re-run on failures, but it is not handling the jobs at the lower granularity level (per study) because he LSF + Java spin up costs are high (so it is worth sending experiments by batches - yet you don't want to be redoing experiments that worked well the last time that you tried that batch).